stories/index.js

```diff
@@ -4,6 +4,8 @@ import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 import Landing from '../src/Components/Landing';
+import Table from '../src/Components/Table';
+import { tableColumns, tableData } from '../src/Services/Constants';
```

Comment:
Do you think that each component should have its own data file?

Reply:
yeah good call. Will update.